src/hooks/useFirebase.js

```diff
@@ -0,0 +1,33 @@
+import { fb } from './../lib/firebase';
+
+export const useFirebase = (collection) => {
```

Comment:
nice idea to create this custom hook 😎

Reply:
I agree. This hook is so cool! Also, nice job creating utilities for full CRUD operations. I'm usually wary of creating things in anticipation of future needs. However, for these, I'm guessing most of them (or something pretty close to them) will be pretty useful. 
Nice job!